index.js

```diff
@@ -216,6 +224,28 @@ function getPostDate(post) {
 	return luxon.DateTime.fromRFC2822(post.pubDate[0], { zone: 'utc' }).toISODate();
 }
 
+function getTags(post) {
+	if (typeof post.category === 'undefined' || post.category === null) {
+		return '[]';
+	}
+	let tagDomains = argv.categoriestotags
+		? ['post_tag']
+		: ['post_tag', 'category'];
+	let tags = post.category.filter(cat => tagDomains.includes(cat.$.domain)).map(tag => tag.$.nicename);
```

Comment:
If a tag and a category are the same and `categoriestotags` is on I believe you'll get dupe tags. Should probably de-dupe.

Reply:
Ready to merge?